tests/fixtures.py

```diff
@@ -25,6 +26,12 @@ def get_test_profile(self):
     def setUp(self):
         app.config['TESTING'] = True
         app.testing = True
+
+        # PYTHONPATH is `pwd` when testing and empty otherwise
+        # using it to determine the project root
+        # either get it from environment variable, or it's one level up from this init file
+        app.project_root = os.environ.get('PYTHONPATH', '') or os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
Since you're able to import `imgee`, do a `dirname` on the `imgee` module instead of on `__file__`. Also, why is this variable being added to the `app` object? Where is it used?